ShinyExpenseTracker/src/ca/ualberta/cs/shinyexpensetracker/models/ExpenseItem.java

```diff
@@ -133,4 +134,12 @@ public void setReceiptPhoto(Bitmap photo) {
 	public Bitmap getReceiptPhoto(){
 		return this.receiptPhoto;
 	}
+
+	public boolean doesHavePhoto() {
+		return hasPhoto;
+	}
```

Comment:
Can this just be a check for `receiptPhoto == null`? Checking and setting a flag manually is a bit unnecessary.